jwql/instrument_monitors/pipeline_tools.py

```diff
@@ -0,0 +1,249 @@
+"""Various utility functions related to the JWST calibration pipeline
+
+Authors
+-------
+
+    - Bryan Hilbert
+
+Use
+---
+
+    This module can be imported as such:
+
+    >>> from jwql.instrument_monitors import pipeline_tools
+    pipeline_steps = pipeline_tools.completed_pipeline_steps(filename)
+ """
+
+from collections import OrderedDict
+import copy
+import numpy as np
+
+from astropy.io import fits
+from jwst import datamodels
```

Comment:
@bhilbert4 It looks like this `datamodels` import isn't actually used in this module.

Reply:
It was...until I decided that the extra speed associated with just using `astropy.fits` was worth it. I'll remove `datamodels`.